extensions/amp-facebook-page/0.1/test/test-amp-facebook-page.js

```diff
@@ -0,0 +1,265 @@
+        tagName: 'AMP-FACEBOOK-PAGE',
+      };
+
+      facebook(win, {
+        href,
+        locale: 'fr_FR',
+        width: 100,
+        height: 100,
+      });
+      const container = doc.body.getElementsByClassName('fb-page')[0];
+      expect(container).not.to.be.undefined;
+      expect(container.getAttribute('data-href')).to.equal(href);
+
+      const script = doc.body.querySelector('script');
+      expect(script.src).to.contain('fr_FR');
+    });
+
+    it('adds fb-page element with data-tabs', () => {
+      const div = document.createElement('div');
+      div.setAttribute('id', 'c');
+      doc.body.appendChild(div);
+      win.context = {
+        tagName: 'AMP-FACEBOOK-PAGE',
+      };
+
+      facebook(win, {
+        href,
+        tabs: 'messages',
+        width: 100,
+        height: 100,
+      });
+      const container = doc.body.getElementsByClassName('fb-page')[0];
+      expect(container).not.to.be.undefined;
+      expect(container.getAttribute('data-href')).to.equal(href);
+      expect(container.getAttribute('data-tabs')).to.equal('messages');
+    });
+
+    it('adds fb-page element with data-hide-cover', () => {
+      const div = document.createElement('div');
+      div.setAttribute('id', 'c');
+      doc.body.appendChild(div);
+      win.context = {
+        tagName: 'AMP-FACEBOOK-PAGE',
+      };
+
+      facebook(win, {
+        href,
+        hideCover: true,
+        width: 100,
+        height: 100,
+      });
+      const container = doc.body.getElementsByClassName('fb-page')[0];
+      expect(container).not.to.be.undefined;
+      expect(container.getAttribute('data-href')).to.equal(href);
+      expect(container.getAttribute('data-hide-cover')).to.equal('true');
+    });
+
+    it('adds fb-page element with data-hide-cta', () => {
+      const div = document.createElement('div');
+      div.setAttribute('id', 'c');
+      doc.body.appendChild(div);
+      win.context = {
+        tagName: 'AMP-FACEBOOK-PAGE',
+      };
+
+      facebook(win, {
+        href,
+        hideCta: true,
+        width: 100,
+        height: 100,
+      });
+      const container = doc.body.getElementsByClassName('fb-page')[0];
+      expect(container).not.to.be.undefined;
+      expect(container.getAttribute('data-href')).to.equal(href);
+      expect(container.getAttribute('data-hide-cta')).to.equal('true');
+    });
+
+    it('adds fb-page element with default attrs', () => {
+      const div = document.createElement('div');
+      div.setAttribute('id', 'c');
+      doc.body.appendChild(div);
+      win.context = {
+        tagName: 'AMP-FACEBOOK-PAGE',
+      };
+
+      facebook(win, {
+        href,
+        width: 100,
+        height: 100,
+      });
+      const container = doc.body.getElementsByClassName('fb-page')[0];
+      expect(container).not.to.be.undefined;
+      expect(container.getAttribute('data-href')).to.equal(href);
+      expect(container.getAttribute('data-tabs')).to.equal('undefined');
```

Comment:
It's not great that many of these are `undefined` --- we should probably fix these to utilize the [documented defaults](https://amp.dev/documentation/components/amp-facebook-page/#attributes_1), but was thinking this might be better in a separate PR so we don't accidentally break something while we introduce tests...